src/ts/utils/dom.ts

```diff
@@ -19,6 +19,20 @@ export function getActiveEditElement(): ValueElement {
     return element as ValueElement;
 }
 
+export function getTopLevelBlockList() {
```

Comment:
I think we talked about this one a bit before, but the "list" name does not really work with returning 1 element

Reply:
changed to `getTopLevelBlocks`